source/blog/2017-10-13-unboxed-roundup-our-links-for-w-c-9th-october-2017.html.markdown

```diff
@@ -0,0 +1,37 @@
+---
+weekly_roundup: true
+title: 'Unboxed Roundup: Our links for w/c 9th October 2017'
+date: '2017-10-13 12:00:00 UTC'
+authors:
+  - 'Neil van Beinum'
+tags: # (Delete as appropriate)
+- Culture
+
+---
+## Sustainable Food Systems at Green Lab - [Martyn E](/people#martyn-evans)
+
+http://greenlab.org/
+
+I met up with my old friend Ande Gregson this week and found out about his new venture Green Lab. It's like a strange experimental potting shed with Ande as resident restless eccentric connecting people, plants, fish, insects and high-tech IOT and fabrication devices. If you've ever fancied fooling around with hydroponics you should get involved.
+
+## Hackers Paradise - [Chris H](/people#chris-holmes)
+
+https://www.hackerparadise.org/
+
+A colleague shared this link in one of our Slack channels this week, organised trips all over the world for developers, designers and entrepreneurs who want to travel while working remotely. 
+I've worked remotely in the past and did take a few small trips but I really like the idea of doing so with a group of like-minded people. The prices vary depending on the location but are around
+$700 per week for a short trip and $500 per week for longer trips so not a budget holiday but something I would consider doing next time I'm working remotely.
+
+## Why Ruby app servers break on macOS High Sierra - [Andrew W](/people#andrew-white)
+
+https://blog.phusion.nl/2017/10/13/why-ruby-app-servers-break-on-macos-high-sierra-and-what-can-be-done-about-it/
+
+If you've upgraded your Mac to High Sierra then you may have noticed some cryptic errors when running Puma or Passenger which is due to changes with the `fork()` system call. In this article by Hongli Lai (author of Passenger), are the reasons why Apple made the changes and what Ruby app server authors are doing to address the problem.
+
+## Track of the Week - Chris S
```

Comment:
I suppose this would normally include a link to the profile but he doesn't have a profile? Also, I just wanted to try to make an inline comment :)

Reply:
You are correct 😄